### DISCUSSION TITLE
other kinds of fan charts?

### DISCUSSION THREAD

**Discussion by dmccallie** [gramps-project/gramps-web#548] [General]
I see the standard fan chart, but don't see any way to show the "descendant" or "2-way" fan charts that exist in the desktop gramps software package.  Is there a way to show these additional fan chart styles?  I looked at the code and only see fanchart.js, so I'm guessing that the other types have not been implemented?  Are there any plans to do so?

**Comment by DavidMStraub** [comment]
Hi! The code for fan charts is separate between Gramps Web & desktop and indeed the descendant and 2-way fan charts haven't been implemented for web yet.
